### DISCUSSION TITLE
How to move all database items

### DISCUSSION THREAD

**Discussion by ItsLeon15** [pterodactyl/panel#2841] [Support]
### Moving the database

_**How would I move the database since it is showing an Error 500 whereas another database works fine? Any help on this would be great!**_

_**I just need to see the servers again?**_

Thanks!

**Comment by ghostdevv** [comment]
What do you mean by move? Move the database to another providor?

**Reply by ghostdevv** [reply]
If so and you have access to phpmyadmin there should be an export tab at the top when you are on your database and then on the new host you can import it back again

**Comment by ItsLeon15** [comment]
Since we are planing to move to a new VPS Host which will have a new database, we want to move the database schems with us, but we do not know how to since it is a terminal view not using PHPMyAdmin.

**Comment by parkervcp** [comment]
using phpmyadmin or mysqldump to get your database are ways to do this but not really a pterodactyl issue.

Be sure to back up your `.env` file in the panel folder.

**Reply by ItsLeon15** [reply]
Thanks, the reason was that we changed the API Key meaning the database caused an error

**Comment by TekExplorer** [comment]
is this the panel's database, or a database host for nodes?

**Reply by ItsLeon15** [reply]
Panel - the API change caused the error
